integration-tests/test-cases/scenario-tests/src/test/java/io/cellery/integration/scenario/tests/petstore/domain/User.java

```diff
@@ -82,39 +70,39 @@ public User(String firstName, String lastName, String address, String userName,
      * Get the user name of user.
      * @return username
      */
-    public String getUserName() {
+    private String getUserName() {
```

Comment:
Do we need this private getters? Can't we use the variable directly?

Reply:
Updated and used the variables directly.